### DISCUSSION TITLE
can argocd ui show objects that are in k8s cluster that were never in git?

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-cd#15064] [General]
can argocd ui show objects that are in k8s cluster that were never in git?

**Comment by davidmontoyago** [comment]
hi @tooptoop4, yes, it does so for namespace-level resources that are allowed in the `AppProject`. You can also display warnings by by configuring `orphanedResources`.

See:
https://argo-cd.readthedocs.io/en/stable/user-guide/orphaned-resources/
https://argo-cd.readthedocs.io/en/stable/operator-manual/project-specification/
